common/enrolment_helper.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime, timedelta
+from logging import getLogger
+from random import randint
+
+from structlog import wrap_logger
+
+from controllers.collection_exercise_controller import create_and_execute_collection_exercise
+from controllers.survey_controller import create_survey, create_classifiers
+
+logger = wrap_logger(getLogger(__name__))
+
+def generate_new_iac_code(context):
```

Comment:
🎨 missing extra line break between the logger and function definition

Reply:
fixed